File: C#_Aplication/Agilize/Tasks.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Agilize
{
    public class Tasks
    {

        public String TaskName {  get; set; }

        public String Description { get; set; }

        public String DateCreation {  get; set; }

        public String DeadLine { get; set; }

        public TaskState CurrentState { get; set; }

        public int Sprint {  get; set; }

        public String EstimatedTime { get; set; }

        public BindingList<Users> TaskMembers { get; set; }

        public Tasks()
        {
            TaskMembers = new BindingList<Users>();
        }

        public Tasks(String taskName, String description, String dateCreation, String deadLine, TaskState currentState, int sprint, string estimatedTime, BindingList<Users> taskMembers)
        {
            TaskMembers = new BindingList<Users>();
            this.TaskName = taskName;
            this.Description = description;
            this.DateCreation = dateCreation;
            this.DeadLine = deadLine;
            this.CurrentState = currentState;
            this.Sprint = sprint;
            this.EstimatedTime = estimatedTime;
            this.TaskMembers = taskMembers;
        }


        override public String ToString()
        {
            return TaskName.ToString();
        }

    }
    public enum TaskState
    {
        BackLog,
        ToDo,
        Doing,
        Pending_Confirmation,
        Done

    }
}

File: Mobile_Aplication/app/src/main/java/com/example/agilize/classes/Tasks.kt
package com.example.agilize.classes

import android.os.Parcel
import android.os.Parcelable


class Tasks(var TaskName: String, var Description: String, var DateCreation: String, var DeadLine: String,
            var CurrentState: Int, var Sprint: Int, var EstimatedTime: String, var TaskMembers: MutableList<Users>):
    Parcelable {
    constructor(parcel: Parcel) : this(
        parcel.readString().toString(),
        parcel.readString().toString(),
        parcel.readString().toString(),
        parcel.readString().toString(),
        parcel.readInt(),
        parcel.readInt(),
        parcel.readString().toString(),
        mutableListOf<Users>().apply {
            parcel.readTypedList(this, Users.CREATOR)
        }
    )

    override fun writeToParcel(parcel: Parcel, flags: Int) {
        parcel.writeString(TaskName)
        parcel.writeString(Description)
        parcel.writeString(DateCreation)
        parcel.writeString(DeadLine)
        parcel.writeInt(CurrentState)
        parcel.writeInt(Sprint)
        parcel.writeString(EstimatedTime)
        parcel.writeTypedList(TaskMembers)
    }

    override fun describeContents(): Int {
        return 0
    }

    companion object CREATOR : Parcelable.Creator<Tasks> {
        override fun createFromParcel(parcel: Parcel): Tasks {
            return Tasks(parcel)
        }

        override fun newArray(size: Int): Array<Tasks?> {
            return arrayOfNulls(size)
        }
    }
}